livesync/tests/queue_test.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+
+from indico_livesync.util import get_excluded_categories, obj_ref
+from indico_livesync.models.queue import LiveSyncQueueEntry, ChangeType
+
+
+CATEGORY_PARENTS = {
+    0: None,
+    1: 0,
+    2: 0,
+    3: 2,
+    4: 3,
+    5: 3
+}
+
+
+def test_excluded_categories(mocker, monkeypatch, db, create_category, dummy_agent):
+    """Test if category exclusions work"""
+
+    # IMPORTANT: `dummy_agent` is actually needed here, since ``LiveSyncQueueEntry.create``
+    # will look for all existing agents in the DB. `db.session.add` is not necessary but
+    # it keeps static checkers happy.
+    db.session.add(dummy_agent)
```

Comment:
You can use `@pytest.mark.usefixtures('dummy_agent')` if you just need to enable a fixture but not actually access it